### ISSUE TITLE
Email notification when Gateway is offline

### ISSUE THREAD

**Issue by jamilbk** [CloudFire-LLC/cloudfire#6180]
As an admin it would be helpful to know when a Gateway is offline.

"Offline" in this case could be defined as a network split with the portal for more than 5 minutes.

**Comment by thomaseizinger** [comment]
We do expose a healthcheck endpoint at `0.0.0.0:8080/healthz`. Perhaps that needs to be documented better so customers can tap into it?

Currently, that health-check doesn't fail if we get partitioned from the portal but that can be changed.

Would that also be acceptable?

**Comment by jamilbk** [comment]
Yeah that may work if the customer has monitoring infra already in place to tie into, but it would be nice to have an email as well just to make things a bit easier
